### PR TITLE
Query to retrieve all edges attached to a node

### PR DESCRIPTION
**Pull request by 3C111**

### CODE REVIEW

File: app/controllers/GenericExplorerController.scala [hunk 1]
```diff
@@ -96,6 +96,25 @@ class GenericExplorerController @Inject() (
     future.map( i => Ok( Json.toJson( i ) ) )
   }
 
+  //Get all edges belonging to a node
+  def retrieveNodeEdges( id: Long ) = ProfileFilterAction( jwtVerifier.get ).async { implicit request =>
```
Comment (leafty) [inline app/controllers/GenericExplorerController.scala]: Can we include filters for in/out? see https://github.com/SwissDataScienceCenter/renga-graph/blob/master/navigation/service/app/controllers/EdgeController.scala#L104-L135 for reference.

Comment (ebiiii) [reply app/controllers/GenericExplorerController.scala]: Indeed could be nice to have. However, currently the edges direction has no real semantic (if you change the edge label you can easily invert the direction). (stores / stored_in).
But to be future-proof, we can add it.

File: test/controllers/GenericExplorerControllerSpec.scala [hunk 2]
```diff
@@ -110,6 +112,46 @@ class GenericExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
...
+      val content = contentAsJson( result ).as[List[PersistedEdge]]
+      val content_ids = for ( i <- content ) yield i.id
+
+      content.length mustBe t.length
+      content_ids.toSet mustBe t_ids.toSet
+    }
+  }
+
+  "The edge retrieval of a node controller" should {
+    "return an empty list if the node with id does not exist" in {
```
Comment (ebiiii) [inline test/controllers/GenericExplorerControllerSpec.scala]: Wouldn't a 404 be more appropriate?

**Comment by 3C111** [comment]
Closed due to travis issues. Will reopen a new one for this issue.

**Pull request merged**